doc/src/site/sphinx/configuration/configuration_properties.rst

```diff
@@ -264,6 +264,16 @@ Internal backend configuration properties
 |                                                    |                |                                                 | settings and other HDFS-related        |
 |                                                    |                |                                                 | configurations.                        |
 +----------------------------------------------------+----------------+-------------------------------------------------+----------------------------------------+
+| ``spark.ext.h2o.internal.clouding.timeout``        | ``0``          | ``setInternalBackendCloudingTimeout(Int)``      | Specifies how long the clouding should |
```

Comment:
`Specifies how long the clouding should last.` -> `Specifies how long the discovering of Spark executors should last.` Clouding up of H2O cluster will happen afterwards, won't it?

Reply:
Yup, good catch!